RHDEVDOCS-3176: Add release notes for 4.6 ZStream (#35882)
openshift/openshift-docs: release_notes/ocp-4-6-release-notes.adoc

* While creating a sample application, the *Developer* perspective creates multiple resources that depend on each other and must be completed in a specific order. Previously, the admission plug-in sometimes could not check one of these resources, preventing the *Developer* perspective from generating the sample application. This issue has been fixed. The code creates the resources in the required order, so creating a sample application is more stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1933666[*BZ#1933666*])

> * Previously, the users were not able to create a knative service as a cluster local from the *Developer* perspective. This issue has now been fixed by changing the label required for making a service cluster local to `'networking.knative.dev/visibility': 'cluster-local'`. The users are now able to create a knative service as cluster local from the *Developer* perspective. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1978159[*BZ#1978159*])

Reviewer: ```suggestion
* Previously, users were not able to create a Knative service as a private service from the *Developer* perspective. This issue has now been fixed, by updating the label `'networking.knative.dev/visibility': 'cluster-local'`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1978159[*BZ#1978159*])
```
Some suggestion to make this one clearer - please always capitalize Knative also

> * Previously, the topology URLs created for deployments using Bitbucket repository in the {product-title} web console did not work if they included a branch name that contained a slash character. This was due to an issue with the Bitbucket API link:https://jira.atlassian.com/browse/BCLOUD-9969[BCLOUD-9969]. The current release mitigates this issue: if a branch name contains a slash, the topology URLs point to the default branch page for the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972694[*BZ#1972694*])

Reviewer: ```suggestion
* Previously, the topology URLs created for deployments using a Bitbucket repository in the {product-title} web console did not work if they included a branch name that contained a slash character. This was due to an issue with the Bitbucket API link:https://jira.atlassian.com/browse/BCLOUD-9969[BCLOUD-9969]. The current release mitigates this issue: if a branch name contains a slash, the topology URLs point to the default branch page for the repository. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972694[*BZ#1972694*])
```
Minor nit, missing "a"

> * Previously, the pipelines created in the git import flow for the private repositories failed to run. This issue has now been fixed by adding the secret name to pipeline ServiceAccount's annotations and adding pipeline-specific annotations to the provided secret. The pipelines for private git repositories now run successfully. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970984[*BZ#1970984*])

Reviewer: ```suggestion
* Previously, pipelines created in the git import flow for private repositories failed to run. This issue has now been fixed by adding the secret name to pipeline `ServiceAccount` annotations, and by adding pipeline-specific annotations to the provided secret. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970984[*BZ#1970984*])
```
Some suggestions - I don't think it's necessary to restate things like " Pipelines for private git repositories now run successfully. " at the end of the issues, because earlier you say "This issue has now been fixed", so it implies that whatever wasn't working is now working.
